dataset/src/test/scala/frameless/functions/AggregateFunctionsTests.scala

```diff
@@ -357,4 +358,52 @@ class AggregateFunctionsTests extends TypedDatasetSuite {
     check(forAll(prop[Vector[Long]] _))
     check(forAll(prop[BigDecimal] _))
   }
+
+  test("corr") {
+    val spark = session
+    import spark.implicits._
```

Comment:
I need it for the `encEv:Encoder[(Int, A, B)]` and the way `session` is handled I can't import `session.implicits._` so I have to reassign it to new val